ARM-wvd-templates/CreateAndProvisionHostPool/CreateHostpoolTemplate.json

```diff
@@ -703,23 +709,18 @@
           "properties": {
             "vMSizeId": "[parameters('vmSize')]",
             "diskType": "[parameters('vmDiskType')]",
-            "vmCustomConfigurationUri": "[if(empty(parameters('customConfigurationTemplateUrl')), '', parameters('customConfigurationTemplateUrl'))]",
-            "vmCustomConfigurationParameterUri": "[if(empty(parameters('customConfigurationParameterUrl')), '', parameters('customConfigurationParameterUrl'))]",
+            "vmCustomConfigurationUri": "[if(empty(parameters('customConfigurationTemplateUrl')), null(), parameters('customConfigurationTemplateUrl'))]",

```

Comment:
why this change?

Reply:
Just want to do it right to set unused value to null in host pool creation scenario.  Although in Add VM scenario, I will update unused properties to empty string for PATCH api call.